### DISCUSSION TITLE
Problem with ExplorerPatcher

### DISCUSSION THREAD

**Discussion by TheSams17** [valinet/ExplorerPatcher#3908] [General]
Hello, everyone. I have been using Explorer patcher for a couple of years. But for some time I have been experiencing a problem. Let me explain: Whenever I start a browser or when I finish watching a youtube video in fullscreen and then exit fullscreen mode, it opens the explorer file for me. I did a test by uninstalling Explorer patcher and it does not, I reinstalled it and the problem returns. It's really annoying that I always have to stand there and ask for the file explorer every time. I am asking for help on how to silvere, thank you.

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.  Please include if you are running x86-64 or Arm.  And what language your windows is in.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.  These should be screen shots.  If you can, please change it to english first.

Please include any other software you have installed that modifies the UI of windows.

Do you have another antivirus installed?

**Comment by TheSams17** [comment]
My windows version: Windows 11 Pro version 23H2 (build SO 22631.4391) Italian x64. Explorer Patcher version: 22621.4317.67.1.
No other software for UI mod. Can i include a current settings export? Avast Premium.

**Comment by pyrates999** [comment]
it might be caused by a corrupt driver: https://answers.microsoft.com/en-us/windows/forum/all/windows-explorer-keep-restarting-when-i-play/4a0a64ab-a835-41cf-8c11-8d091fcc8d2a

https://www.reddit.com/r/MicrosoftEdge/comments/1bs4knb/explorerexe_restarting_after_watching_some/

try reinstalling your gpu drivers.

**Comment by TheSams17** [comment]
Hi all, it seems that I solved the problem by uninstalling the GPU drivers. Mode for uninstalling: best in safe mode with DDU, a program suitable for removing GPU and audio drivers. Thanks for the help.
